openapi/mx_platform_api.yml

```diff
@@ -779,6 +779,51 @@ components:
         pagination:
           "$ref": "#/components/schemas/PaginationResponse"
       type: object
+    CreditCardProduct:
+      properties:
+        annual_fee:
+          example: 45.00
```

Comment:
is there a reason to leave the `type` from these first few fields?

Reply:
Just keeping it in sync with what is in the private repo. I thought it odd they were missing as well but figured I could clean that up in my other docs ticket.